### PR TITLE
#37 Send page: Add all balance button

### PR DESCRIPTION
**Pull request by bodogaii**

### CODE REVIEW

File: src/app/pages/send/send.component.tsx [hunk 1]
```diff
@@ -229,4 +246,19 @@ export class SendPage extends Component<IProps, IState> {
...
+        this.props.account
+            .getBalance()
+            .then(balance => {
+                this.setState({
+                    balance: this.props.account.utils.balanceToStd(balance)
+                });
+            })
+            .catch(() => {
+                this.setState({
+                    balance: 'error'
```
Comment (krisboit) [inline src/app/pages/send/send.component.tsx]: as an error handling here we should hide the button if something went wrong on getBalance

File: src/app/translations/en.ts [hunk 2]
```diff
@@ -55,7 +55,7 @@ export const translation: ITranslations = {
                 generic:
                     'Sorry, there was an error posting your transaction to blockchain. Try again or come back later.'
             },
-
+            addAllBalance: 'ADD ALL BALANCE',
```
Comment (krisboit) [inline src/app/translations/en.ts]: buttons already have upercase transformation from css you can use here `Add all balance`

File: src/app/pages/send/send.component.tsx (hunk 1 above)
Comment (bodogaii) [reply src/app/pages/send/send.component.tsx]: Done

File: src/app/translations/en.ts (hunk 2 above)
Comment (bodogaii) [reply src/app/translations/en.ts]: Done

**Pull request closed without merging**